static/src/javascripts/projects/common/views/svg.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+export default (
```

Comment:
We try to avoid `export default` entirely. Instead we want to use named exports, whenever it's possible, so that we can `import {svg} from 'views/svg'`. Makes it easier to refactor.

In this case it means, that you'd have to refactor AMD imports to `svg.svg()`. It's ugly for now, but will be resolved through the whole ES6 conversion process ...